Dockerfile.prod

```
@@ -62,7 +62,8 @@ RUN set -eux; \
 # Switch to non-root user defined in httpd image
 USER www-data
 
-ENV API_URL="/inventory-management-system-api"
+ENV IMS_API_URL="/inventory-management-system-api"
+ENV STORAGE_API_URL = "/object-store-api"
```

```suggestion
ENV STORAGE_API_URL = "/object-storage-api"
```
Perhaps it should be OBJECT_STORAGE_API_URL for consistency unless we started using the acronym, but not sure OS_API_URL is very clear.


Worth noting that from this PR onwards we cannot update dev to point to this branch until docker-ims is updated as well.

---

OBJECT_STORAGE_API_URL is to long. STORAGE_API_URL is fine for now

---

There is no limit (we have much longer ones in the backend due to the way pydantic config works) but I agree IMS written out fully is unnecessary, so happy with OS_API for consistency.